docker-example/Dockerfile

```diff
@@ -25,6 +24,9 @@ RUN apt-get update && apt-get install -y \
     nodejs \
     unzip \
     && rm -rf /var/lib/apt/lists/*
+RUN curl -fsSL https://bun.sh/install | bash 
```

Comment:
we should definitely specify the version of bun to install for compatibility

Reply:
You probably can get rid of Bun install because we install it automatically install it for the user if they don't have it

Reply:
> You probably can get rid of Bun install because we install it automatically install it for the user if they don't have it

if you delete `RUN curl -fsSL https://bun.sh/install | bash` , you will meet `Cannot'find /pynecone/.bun` error !